content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -294,6 +294,38 @@ See [topology.kubernetes.io/zone](#topologykubernetesiozone).
 
 {{< note >}} Starting in v1.17, this label is deprecated in favor of [topology.kubernetes.io/zone](#topologykubernetesiozone). {{< /note >}}
 
+### pv.kubernetes.io/bind-completed {#pv-kubernetesiobind-completed}
+
+Example : `pv.kubernetes.io/bind-completed: "yes"`
+
+Used on: PersistentVolumeClaim
+
+It indicates that the lifecycle of the PVC has passed through the initial setup. This information changes how we interpret some observations of the state of the objects. Value of this Annotation does not matter.
```

Comment:
How about:
```suggestion
When this annotation is set on a PersistentVolumeClaim (PVC), that indicates that the
lifecycle of the PVC has passed through initial binding setup. When present, that
information changes how the control plane interprets the state of PVC objects.
The value of this annotation does not matter to Kubernetes.
```

Reply:
Thanks for the suggestions. I'll make the changes.